src/main/java/net/javadiscord/javabot/util/GuildUtils.java

```diff
@@ -2,43 +2,18 @@
 
 import net.dv8tion.jda.api.entities.Guild;
 import net.dv8tion.jda.api.entities.Member;
+import org.jetbrains.annotations.NotNull;
 
-import java.util.List;
 
 /**
  * Utility class for various things.
  */
+@Deprecated(forRemoval = true)
```

Comment:
Remove all usages of this class and then remove the class instead of deprecating it.